### PR TITLE
Makefile: distinguish stable and rolling

### PR DESCRIPTION
**Pull request by runn1ng**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,30 @@
+# for stable (numbered) versions:
+# make dist-stable-v1 && make sync-stable-v1
+# the result is on connect.trezor.io/1
+
+# for rolling versions:
+# make dist-rolling-claim-bch && make sync-rolling-claim-bch
+# the result is on connect.trezor.io/claim-bc-ce179c9
```
Comment (prusnak) [inline Makefile]: typo, should be claim-bch not claim-bc

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,30 @@
...
+# ce179c9 is first 7 letters from the hash of the commit (what github etc shows)
+
 clean:
 	rm -rf dist/
 
-dist-%:
-	git fetch --tags
+dist-rolling-%:
+	git tag -d `git tag` #delete all local tags to prevent use of tag
+	# git fetch --tags
```
Comment (prusnak) [inline Makefile]: remove this line

File: Makefile [hunk 3]
```diff
@@ -1,9 +1,30 @@
...
+	git fetch
+	git checkout $*
+	make .copy-$*-`git rev-parse --short HEAD`
+
+dist-stable-v%:
+	git tag -d `git tag` #delete all local tags to prevent use of tag
+	# git fetch --tags
```
Comment (prusnak) [inline Makefile]: same as above

File: Makefile (hunk 1 above)
Comment (runn1ng) [inline Makefile]: This was meant as an example :) but ok

**Pull request merged**